charts/incubator/bar-assistant/values.yaml

```diff
@@ -0,0 +1,47 @@
+image:
+  repository: barassistant/server:v3
+  pullPolicy: IfNotPresent
+  tag: 
+
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          env:
+            MEILI_MASTER_KEY: masterKey-make-it-long-for-security
+            BASE_URL: http://localhost:3010
+            MEILISEARCH_URL: ${BASE_URL}/search
+            API_URL: ${BASE_URL}/bar
+
```

Comment:
Need to add the other containers here. Right now your chart only consists of one container